cwms-data-api/src/test/java/cwms/cda/api/StateControllerTestIT.java

```diff
@@ -56,4 +57,24 @@ void test_state_catalog()  {
             .body("[0].name", equalTo("Unknown State or State N/A"))
             .body("[0].state-initial", equalTo("00"));
     }
+
+    @Test
+    void test_state_has_ETag_and_Cache_Control()  {
+        String matcher;
+        given()
+            .log().ifValidationFails(LogDetail.ALL,true)
```

Comment:
good test, ETag shows up, Cache-Control does not.

FYI had to merge with the apikey fix. one of the things I fixed there was preventing the proper retrieval of the datasource.

Reply:
Oh. I think I know whats going on.   At least I hope I do.  In my testing I saw that it adds it as cache-control all lowercase.   according to SO  http/2.0 says that the header field names must be converted to lower case.  
https://stackoverflow.com/a/41169947/249623

Reply:
I still haven't been able to get my IDE to run things.  I'm working on something else at the moment but its annoying.

Reply:
The test I ran printed the headers, the cache-control header wasn't present for any casing... which was surprising I didn't see anything wrong with the code you made.

What kind of errors is you're IDE giving you? Karl had some issues with Intellij and Ant due to a version mismatch. Might be something similar since we need to fix the build to allow a java11+ compiler to work even if we are still targetting 8.

Reply:
Also, It's reasonable to put those header expectations in the test_state_catalog. Seems a waste of resources to just make the request again.

Though an interesting test would be to make the request again right after and see if it's cached, but I'm not sure if the restassured client actually does that.